docs/content.zh/docs/dev/table/types.md

```diff
@@ -1179,26 +1127,21 @@ DataTypes.MULTISET(t)
 {{< /tab >}}
 {{< /tabs >}}
 
-The type can be declared using `MULTISET<t>` where `t` is the data type
-of the contained elements.
 
-`t MULTISET` is a synonym for being closer to the SQL standard. For example, `INT MULTISET` is
-equivalent to `MULTISET<INT>`.
+可以使用 `MULTISET<t>` 声明类型，其中 `t` 是数据类型 所包含的元素。
+`t MULTISET` 是更接近 SQL 标准的同义词。 例如，`INT MULTISET` 相当于`MULTISET<INT>`。 
 
-#### `ROW`
 
-Data type of a sequence of fields.
+#### `ROW`
 
-A field consists of a field name, field type, and an optional description. The most specific type
-of a row of a table is a row type. In this case, each column of the row corresponds to the field
-of the row type that has the same ordinal position as the column.
 
-Compared to the SQL standard, an optional field description simplifies the handling with complex
-structures.
+字段序列的数据类型。
 
-A row type is similar to the `STRUCT` type known from other non-standard-compliant frameworks.
+字段由字段名称、字段类型和可选描述组成。 最具体的类型 表的一行是行类型。 在这种情况下，行的每一列对应于字段 与列具有相同序号位置的行类型。
```

Comment:
The previous work of the contributor is not clear enough. And it also doesn't correct grammarly.